VisualStudio/Config.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace VisualStudio
+{
+    class Config
+    {
+        readonly string section;
+        readonly IProcessRunner runner;
+
+        public Config(string section, IProcessRunner runner)
+        {
+            this.section = section;
+            this.runner = runner;
+        }
+
+        public async Task SetValueAsync(string name, string value, bool setGlobal = false) =>
+            await runner.StartAsync("dotnet", "config", $"{section}.{name}", "" + value + "", setGlobal ? "--global" : "--local");
```

Comment:
Rather than starting an external tool, this should use the [dotnet-config-lib](https://www.nuget.org/packages/dotnet-config-lib) API directly instead.